pagerduty/komand_pagerduty/actions/get_on_call/action.py

```diff
@@ -0,0 +1,43 @@
+import komand
+from .schema import GetOnCallInput, GetOnCallOutput, Input, Output, Component
+# Custom imports below
+import pypd
+from komand.exceptions import PluginException
+
+
+class GetOnCall(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='get_on_call',
+                description=Component.DESCRIPTION,
+                input=GetOnCallInput(),
+                output=GetOnCallOutput())
+
+    def run(self, params={}):
+        # TODO: Implement run function
+        """Get list of users"""
```

Comment:
```suggestion
        # Get list of users
```
Don't want to use a docstring comment here, those are best for function documentation